src/nodes/Texturing/Texture.js

```diff
@@ -39,7 +39,7 @@ x3dom.registerNodeType(
             this.addField_SFBool( ctx, "hideChildren", true );
 
             this._video = null;
-            this._intervalID = 0;
+            this._intervalID = null;
```

Comment:
The id is actually a numerical value.

https://developer.mozilla.org/en-US/docs/Web/API/setInterval#return_value

Reply:
Yes, setInterval() returns a non-zero value.

Reply:
I mean initialising it to 0 seems more appropriate rather than null.

Reply:
In fact, there is no difference between 0 and null, I prefer null, because it clearly tells me that there is no interval, and 0 may cause misunderstanding, making people mistakenly think that interval 0 is an existing interval.

Reply:
ok but note 0 === null
false